library/include/modules/Persistence.h

```diff
@@ -213,5 +213,7 @@ namespace DFHack
         // Fills the vector with references to each persistent item with a key that is
         // equal to the given key.
         DFHACK_EXPORT void getAllByKey(std::vector<PersistentDataItem> &vec, int entity_id, const std::string &key);
+        // Returns the tickcount of the most recent save.
+        DFHACK_EXPORT uint32_t getSaveDur();
```

Comment:
I don't think `getSaveDur` is an appropriate name. ~~How about `getLastLoadSaveTickCount()`? It's long, but it's also much more specific. The name needs to communicate that it is updated on load or save, and that it talks in the same units as getTickCount().~~ see comment in Persistence.cpp

This information should also be in the documentation in `Lua API.rst`.